core/server/master/src/main/java/alluxio/master/metastore/heap/HeapInodeStore.java

```diff
@@ -78,6 +81,25 @@ public void remove(Long inodeId) {
     mInodes.remove(inodeId);
   }
 
+  @Override
+  public void writeNewInode(MutableInode<?> inode) {
+    mInodes.compute(inode.getId(), (k, existingInode) -> {
+      if (existingInode != null && !existingInode.getName().equals(inode.getName())) {
+        LOG.error(
+            "[InodeTreeCorruption] writing inode {} with id {}, parent id {} "
```

Comment:
```suggestion
            "[InodeTreeCorruption] overwriting inode {} with id {}, parent id {} "
```

Reply:
this is actually writing not overwriting. i changed the desc a bit better